jjb/volume/edgex-volume.yaml

```diff
@@ -25,17 +25,20 @@
 
     jobs:
       - '{project-name}-{stream}-verify-docker'
-      - '{project-name}-{stream}-merge-docker'
-      - '{project-name}-{stream}-release-version-docker-daily-no-sonar'
+      - '{project-name}-{stream}-merge-docker':
+          custom_triggers:
```

Comment:
The custom triggers in this file need to be removed.

Reply:
Done 👍